InterGalacticCurrencyConvertor/src/main/java/materials/MaterialFactory.java

```diff
@@ -0,0 +1,70 @@
+package main.java.materials;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+import main.java.IgnConversionErrors;
+import main.java.IgnConversionException;
+
+/**
+ * Keeps track of different types of materials that can be sold InterGalactically.
+ */
+public final class MaterialFactory {
+  private final HashMap<String, Material> mMtlMap  = new HashMap<>();
```

Comment:
rename mMtlMap to mValidMaterials or something...

Reply:
done